src/connectors/WalletConnectV2Connector.ts

```diff
@@ -141,6 +141,6 @@ export class WalletConnectV2Connector extends AbstractConnector {
       throw new Error('Provider is undefined')
     }
 
-    this.deactivate()
+    return this.deactivate()
```

Comment:
Is returning necessary? why not just await it?

Reply:
That's because there's nothing to be awaited for, it can just return the promise.